packages/module/src/behavior/withContextMenu.tsx

```diff
@@ -10,6 +10,7 @@ type Reference = React.ComponentProps<typeof ContextMenu>['reference'];
 export interface WithContextMenuProps {
   onContextMenu?: (e: React.MouseEvent) => void;
   contextMenuOpen?: boolean;
+  hideContextMenuKebab?: boolean;
```

Comment:
I don't think we need to add this.

Reply:
`NodeLabel` raises an error unless `hideContextMenuKebab` is declared within `NodeLabelProps` - added it [here](https://github.com/patternfly/react-topology/pull/250/commits/3fcecf29999b68b2340265b50b38c353f4262557#diff-4d112d4ea261ae551fe98a9c181b60c2ea3bcd8b61cb702780af315f7f8f72abR46).